pkg/network/config/config_test.go

```diff
@@ -1557,7 +1586,7 @@ system_probe_config:
 		aconfig.ResetSystemProbeConfig(t)
 		cfg := modelCfgFromYAML(t, `
```

Comment:
What's the difference between this and `configurationFromYAML`?

Reply:
yes, indeed the modelCFG returns the viper interface, while configurationFromYaml returns the actual ORM we use in the package

Reply:
Are these interchangeable for the tests?

Reply:
No, `TestProcessServiceInference` and `TestProcessServiceInferenceWindows` are not "supported" as we didn't expose `process_service_inference` subtree in our ORM, as the only place accessing this configuration in process-agent

Reply:
To make the change, we'll need to create entries in the ORM just for the test

Reply:
Maybe a comment would be helpful or a better name for the function so that this confusion doesn't happen again